exaudfclient/base/build_local.sh

```diff
@@ -1,2 +1,5 @@
+#!/bin/bash
+
+#shellcheck disable=SC1091
 source .env
-bash build.sh $*
\ No newline at end of file
+bash build.sh "$@"
```

Comment:
Reason:
"                    ^-- SC2048: Use "$@" (with quotes) to prevent whitespace problems.
                    ^-- SC2086: Double quote to prevent globbing and word splitting.
"

Reply:
Should be fine